src/Analyzers/CSharp/Tests/MakeStatementAsynchronous/CSharpMakeStatementAsynchronousCodeFixTests.cs

```diff
@@ -2,226 +2,240 @@
+        foreach (var ({|CS8130:k|}, {|CS8130:l|}) in {|CS8414:collection|}) { }
     }
 }",
-IAsyncEnumerable + @"
+                FixedCode = @"
 class Program
 {
-    void M(System.Collections.Generic.IAsyncEnumerable<(int, int)> collection)
+    async void M(System.Collections.Generic.IAsyncEnumerable<(int, int)> collection)
     {
         await foreach (var (i, j) in collection) { }
         await foreach (var (k, l) in collection) { }
     }
-}", parameters: s_asyncStreamsFeature);
+}",
+                ReferenceAssemblies = ReferenceAssemblies.Net.Net60
+            }.RunAsync();
         }
 
         [Fact]
         public async Task FixAllUsingStatement()
         {
-            await TestInRegularAndScript1Async(
-IAsyncEnumerable + @"
+            await new VerifyCS.Test()
+            {
+                TestCode = @"
 class Program
 {
-    void M(System.IAsyncDisposable disposable)
+    async void M(System.IAsyncDisposable disposable)
     {
-        using (var i = {|FixAllInDocument:disposable|}) { }
-        using (var j = disposable) { }
+        using ({|CS8418:var i = disposable|}) { }
+        using ({|CS8418:var j = disposable|}) { }
     }
 }",
-IAsyncEnumerable + @"
+                FixedCode = @"
 class Program
 {
-    void M(System.IAsyncDisposable disposable)
+    async void M(System.IAsyncDisposable disposable)
     {
         await using (var i = disposable) { }
         await using (var j = disposable) { }
     }
-}", parameters: s_asyncStreamsFeature);
+}",
+                ReferenceAssemblies = ReferenceAssemblies.Net.Net60
+            }.RunAsync();
         }
 
         [Fact]
         public async Task FixAllUsingDeclaration()
         {
-            await TestInRegularAndScript1Async(
-IAsyncEnumerable + @"
+            await new VerifyCS.Test()
+            {
+                TestCode = @"
 class Program
 {
-    void M(System.IAsyncDisposable disposable)
+    async void M(System.IAsyncDisposable disposable)
     {
-        using var i = {|FixAllInDocument:disposable|};
-        using var j = disposable;
+        {|CS8418:using var i = disposable;|}
+        {|CS8418:using var j = disposable;|}
     }
 }",
-IAsyncEnumerable + @"
+                FixedCode = @"
 class Program
 {
-    void M(System.IAsyncDisposable disposable)
+    async void M(System.IAsyncDisposable disposable)
     {
         await using var i = disposable;
         await using var j = disposable;
     }
-}", parameters: s_asyncStreamsFeature);
+}",
+                ReferenceAssemblies = ReferenceAssemblies.Net.Net60
+            }.RunAsync();
         }
 
         [Fact]
         public async Task FixForeach()
         {
-            await TestInRegularAndScript1Async(
-IAsyncEnumerable + @"
+            await new VerifyCS.Test()
+            {
+                TestCode = @"
 class Program
 {
-    void M(System.Collections.Generic.IAsyncEnumerable<int> collection)
+    async void M(System.Collections.Generic.IAsyncEnumerable<int> collection)
```

Comment:
Is the change to test input intended?

Reply:
Yes, to avoid compilation error about `await` in non-`async` method after fix. This is not the main points of these tests